doc/administrator/editing.rst

```diff
@@ -53,6 +53,22 @@ To be editable, a layer should satisfy the following requirements:
        }
    }
 
+7. The supported column types are:
+
+      * ``integer``
+      * ``numeric``
+      * ``character varying``
+      * ``text``
+      * ``boolean``
+      * ``date``
+      * ``time without time zone``
+      * ``timestamp without time zone``
+
+   Note that the following types will not working correctly with the timezone:
```

Comment:
The sentence is grammatically incorrect,
but more importantly: it is not clear why a type "with time zone" will not work correctly with the timezone. Is it a bug in GMF? Or what exactly is not working?

Reply:
GeoMapFish first implementation uses the types `... without time zone` then the other one needs a specific development....

Reply:
So are we saying the following?
"Note that the following timezone related types are currently not supported:"

Reply:
I don't understand your sentences...
We provide a list of types not of timezones...

Reply:
Ok so:

```suggestion"
   Note that the following timezone aware types are currently not supported:
```